bin/docker-entrypoint.sh

```diff
@@ -2,36 +2,55 @@
 
 set -eo pipefail
 shopt -s nullglob
-
-if [[ ! $INIT_SCRIPTS_PATH ]]
-then
-  # FIXME: deprecate and use /etc/localstack/init/ready.d
-  INIT_SCRIPTS_PATH=/docker-entrypoint-initaws.d
-fi
 if [[ ! $EDGE_PORT ]]
 then
   EDGE_PORT=4566
 fi
 
-# FIXME: deprecation path for legacy directories
+# FIXME: remove with 2.0
+# the Dockerfile creates .pro-version file for the pro image. When trying to activate pro features with any other
+# version, an error is printed.
+if [[ $LOCALSTACK_API_KEY ]] && [[ ! -f /usr/lib/localstack/.pro-version ]]; then
+    echo "WARNING"
+    echo "============================================================================"
+    echo "  It seems you are using the LocalStack Pro version without using the"
+    echo "  dedicated Pro image."
+    echo "  Future versions will only support running LocalStack Pro with the"
+    echo "  dedicated image."
+    echo "  To fix this warning, use localstack/localstack-pro instead."
+    echo ""
+    echo "  See: https://github.com/localstack/localstack/issues/7257"
+    echo "============================================================================"
+    echo ""
+elif [[ -f /usr/lib/localstack/.light-version ]] || [[ -f /usr/lib/localstack/.full-version ]]; then
+    echo "WARNING"
+    echo "============================================================================"
+    echo "  It seems you are using a deprecated image (localstack/localstack-light"
+    echo "  or localstack/localstack-full)."
+    echo "  These images are deprecated and will be removed in the future."
+    echo "  To fix this warning, use localstack/localstack instead."
```

Comment:
```suggestion
    echo "  To fix this warning, use localstack/localstack or localstack/localstack-pro instead."
```

Reply:
Actually this is correct as it is:
- If a `LOCALSTACK_API_KEY` is set, but the pro image is not used: Tell the user to use the `localstack-pro` image.
- **Else If** the light or full images are shown: Tell the user to use the `localstack/localstack` image.